npm/pkg/dataplane/ipsets/ipsetmanager.go

```diff
@@ -316,6 +341,9 @@ func (iMgr *IPSetManager) AddToLists(listMetadatas, setMetadatas []*IPSetMetadat
 		// 3. add all members to the list
 		for _, memberMetadata := range setMetadatas {
 			memberName := memberMetadata.GetPrefixName()
+			if memberName == "" {
+				continue
```

Comment:
can we send an error log when member name is empty? Must be an error in translation right?